app/views/api/v1/registrations/create.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.user @resource, partial: 'info', as: :user
```

Comment:
Maybe you could use a single view named `show` for the create and update methods